### PR TITLE
feat: add 华东理工继续教育学院新闻公告;

### PR DESCRIPTION
**Pull request by jialinghui**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/ecust-jxjy/news/
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [ ] Documentation
  - [x] CN
  - [ ] EN
- [x] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/university.md [hunk 1]
```diff
@@ -1044,6 +1044,10 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
 
 <Route author="sushengmao" example="/ecustyjs" path="/ecustyjs" />
 
+### 华东理工继续教育学院新闻公告
+
+<Route author="jialinghui" example="/news" path="/news" />
```
Comment (TonyRL) [inline docs/university.md:1049]:
```suggestion
<Route author="jialinghui" example="/ecust-jxjy/news" path="/ecust-jxjy/news" />
```

File: lib/v2/ecust-jxjy/news.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            const description = await ctx.cache.tryGet(link, async () => {
+                const result = await got.get(link);
+
+                const $ = cheerio.load(result.data);
+
+                return $('#articlepanel').html();
+            });
+            return {
+                title: item.find('a').text().trim(),
+                pubDate: new Date(date(item.find('dd').text())),
```
Comment (TonyRL) [inline lib/v2/ecust-jxjy/news.js:33]: Please use `parseDate()`.
Docs: https://docs.rsshub.app/joinus/pub-date.html

File: lib/v2/ecust-jxjy/radar.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'jxjy.ecust.edu.cn': {
+        _name: '华东理工继续教育学院',
+        '.': [
+            {
+                title: '新闻公告',
+                docs: 'https://docs.rsshub.app/university.html#hua-dong-li-gong-ji-xu-jiao-yu-xue-yuan',
+                source: ['/'],
+                target: '/news',
```
Comment (TonyRL) [inline lib/v2/ecust-jxjy/radar.js:9]:
```suggestion
                target: '/ecust-jxjy/news',
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Comment by TonyRL** [comment]
I would suggest using the route `/ecust/jxjy/news` instead of `/ecust-jxjy/news` for better route management.

**Comment by jialinghui** [comment]
> I would suggest using the route `/ecust/jxjy/news` instead of `/ecust-jxjy/news` for better route management.

Thanks for the suggestion, but I think `/ecust-jxjy/news` is appropriate

**Pull request merged**